src/adapter/http/src/smart_protocol/axum_server_push_protocol.rs

```diff
@@ -74,7 +74,20 @@ impl AxumServerPushProtocolInstance {
                 tracing::debug!("Push process success");
             }
             Err(e) => {
-                tracing::debug!("Push process aborted with error: {}", e);
+                if let PushServerError::Internal(int_err) = &e {
+                    if let Err(write_err) = axum_write_close_payload::<DatasetPushResponse>(
+                        &mut self.socket,
+                        Err(DatasetPushRequestError::Internal(TransferInternalError {
+                            phase: int_err.phase.clone(),
+                            error_message: "Internal error".to_string(),
+                        })),
+                    )
+                    .await
+                    {
+                        tracing::error!("Failed to send error to client with error: {}", write_err);
+                    };
+                }
+                tracing::error!("Push process aborted with error: {}", e);
```

Comment:
It's best to avoid formatting strings into error messages and use structured approach instead. This will let us easier filter logs by error types.

We should follow the same format as in [other APIs](https://github.com/kamu-data/kamu-cli/blob/90dd02d8fd37d68a99cfac625f5cf41f431b5b75/src/utils/http-common/src/api_error.rs#L140-L145):
```rust
tracing::error!(
  error = ?err,
  error_msg = %err,
  "Push process aborted with error",
);
```

The code above is not ideal, but it would be consistent with other places, and via `?err` it will include the full debug representation of the error, so that we'll see the whole error chain and backtraces.